### PR TITLE
Add Aria2 Configuration Options

### LINKED ISSUE #198: Make aria2 configurable
@fsobolev I believe having the ability to configure both `-x` and `-k` by the user would be really convenient

_Originally posted by @VarLad in https://github.com/NickvisionApps/TubeConverter/issues/29#issuecomment-1514435164_

### PR DESCRIPTION
**Pull request by nlogozzo**
Closes #198

### CODE REVIEW

**Review by DaPigGuy — APPROVED** [review]

File: NickvisionTubeConverter.Shared/Models/Download.cs [hunk 1]
```diff
@@ -221,9 +223,11 @@ public void Start(bool useAria, bool embedMetadata, string? cookiesPath, Localiz
                     dynamic ariaDict = new PyDict();
                     dynamic ariaParams = new PyList();
                     ariaParams.Append(new PyString($"--max-overall-download-limit={(_limitSpeed ? _speedLimit : 0)}K"));
-                    ariaParams.Append(new PyString("--allow-overwrite=true"));
+                    ariaParams.Append(new PyString($"--allow-overwrite={_overwriteFiles}"));
```
Comment (fsobolev) [inline NickvisionTubeConverter.Shared/Models/Download.cs:226]: For aria `allow-overwrite` should always be true to be able to restart downloads.
Also I will improve UI a bit.

**Review by fsobolev — CHANGES_REQUESTED** [review]

File: NickvisionTubeConverter.Shared/Models/Configuration.cs [hunk 2]
```diff
@@ -95,6 +90,8 @@ public Configuration()
         RunInBackground = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
         SpeedLimit = 1024;
         UseAria = false;
+        AriaMaxConnectionsPerServer = 16;
```
Comment (nlogozzo) [inline NickvisionTubeConverter.Shared/Models/Configuration.cs:93]: According to this: https://aria2.github.io/manual/en/html/aria2c.html

The default is 1.

Comment (fsobolev) [reply NickvisionTubeConverter.Shared/Models/Configuration.cs:93]: yt-dlp overwrites the default to 16 https://github.com/NickvisionApps/TubeConverter/issues/29#issuecomment-1514422019

**Comment by fsobolev** [comment]
* Added reset buttons to aria settings
* Set -x to be 16 by default, same as yt-dlp's default (https://github.com/NickvisionApps/TubeConverter/issues/29#issuecomment-1514422019)
* Changed MB -> MiB in setting description (according to [aria docs](https://aria2.github.io/manual/en/html/aria2c.html#cmdoption-k), 1M = 1024K, which means it's MiB)

File: NickvisionTubeConverter.Shared/Models/Configuration.cs (hunk 2 above)
Comment (nlogozzo) [reply NickvisionTubeConverter.Shared/Models/Configuration.cs:93]: Ah ok :)

**Pull request merged**